lib/datapackage.py

```diff
@@ -60,13 +60,27 @@ def create_json():
                       'header_rows (int)']  # int
 
             for i in range(0, len(d_opts)):
```

Comment:
Two ideas that may help to refactor:

 

1. 

I think we can use a dictionary in this case to avoid looping several times,
The dictionary could have this data structure  
`d_opts = { 'key' : ("prompt mesage","default-value"), ....}`

```
d_opts = { 
	'replacecoloms': ("replace_columns (separated by \';\', with comma-separated values)", ), 
	'nulls': ('nulls (separated by \';\')',)
	'delimiter': ('delimiter',)
	'do_not_bulk_insert': ('do_not_bulk_insert (bool = True/False)', 'False')
	'contains_pk': ('contains_pk (bool = True/False)',  'False')
 }

```

so while adding table you read up the d_opts and populate them based on user's input otherwise take the default.

2. 

we can create smaller functions to be specific like `get_replace_colums()`. The main idea is that if we want to debug something or add some new d_opts keyword like file_encoding, it becomes more easy